### PR TITLE
Rework the introductory content

### PR DESCRIPTION
**Pull request by mbelton-buildkite**
This change improves the docs alongside the new in-app onboarding (starter pipeline):

- Updates the Getting started guide to acknowledge the in-app flow. Along with a general uplift providing more context.
- Adds a new page to encourage people to move beyond canned examples and create a real pipeline for their own code/use case. This new page goes after the Getting started guide and after the in-app onboarding.

### CODE REVIEW

**Comment by mbelton-buildkite** [comment]
FYI, @olyism @clbarrell this is the Getting started PR. If you have time, feel free to leave any final feedback/check it works. 😊

File: pages/tutorials/getting_started.md [hunk 1]
```diff
@@ -1,61 +1,118 @@
...
+<%= image "hybrid-saas-architecture.svg", alt: "Shows the hybrid architecture combining a SaaS platform with your infrastructure" %>
+
+The diagram shows that Buildkite provides a web interface, handles integrations with third-party tools, and offers APIs and webhooks. By design, sensitive data, such as source code and secrets, remain within your environment and are not seen by Buildkite. This decoupling provides flexibility and security as you maintain control over the build environment and agent scaling while Buildkite manages the coordination, scheduling, and web interface.
+
+## Install and run an agent
+
+The program that executes work is called an _agent_ in Buildkite. An agent is a small, reliable, and cross-platform build runner that connects your infrastructure to Buildkite. It polls Buildkite for work, runs jobs, and reports results. You can install agents on local machines, cloud servers, or other remote machines. You need at least one agent to run builds.
+
+>📘 Already running an agent
+> If you're already running an agent, skip this section.
```
Comment (dannymidnight) [inline pages/tutorials/getting_started.md:40]: Can we link this directly to the next relevant section?

**Review by dannymidnight — APPROVED** [review]
Awesome work @mbelton-buildkite. It all reads really well.

File: pages/tutorials/getting_started.md (hunk 1 above)
Comment (mbelton-buildkite) [reply pages/tutorials/getting_started.md:40]: Good idea!

```suggestion
> If you're already running an agent, skip to the [next step](#create-a-pipeline).
```

**Pull request merged**